plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
Words like global, general, all, overall fail to convey enough meaning. They all convey that it is unbounded in the field of subject without saying what the field of subject is. So in contrast to printer specific settings these global settings are not bounded to any application of the material.

The problem is that variable names should be self explanatory, while 'global' only has meaning wrt 'printer specific'. 

I don't know. Maybe this is just a philosophical excerpt.